InnerEye/ML/visualizers/plot_cross_validation.py

```diff
@@ -182,6 +177,13 @@ def get_short_name(self, run_or_id: Union[Run, str]) -> str:
                 self.short_names[run_or_id] = run_id
         return self.short_names[run_id]
 
+    @property
+    def execution_mode_to_download(self) -> List[ModelExecutionMode]:
```

Comment:
This could be a method that you can override separately in scalar config and segmentation config?

Reply:
Not sure that's worth it. And maybe we do actually want to download also training metrics for segmentation, if they are available.

Reply:
Metrics are always empty for training in segmentation it seems. I guess the reason is that it would be heavy to compute all of them?

Reply:
Overriding is not worth it? we are essentially doing the similar logic with the if right? I guess if we have a class hierarchy like this it might make sense to use it for these aspects perhaps?

Reply:
I don't know for me this attribute should be an attribute of `PlotCrossValConfig` not of the model itself as it merely concerns the behavior within the cross-validation processing